.github/workflows/make.yml

```
@@ -13,11 +13,13 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - run: sudo apt-get -q update
-      - run: sudo apt-get -qq upgrade
-      - run: sudo apt-get -qq install gettext python3 python3-pip python3-babel python3-boto3 python3-jinja2
-                                      python3-numpy python3-pillow python3-pycurl python3-tornado
+      - run: sudo DEBIAN_FRONTEND="noninteractive" apt-get -qq --no-install-recommends dist-upgrade
+      - run: sudo DEBIAN_FRONTEND="noninteractive" apt-get -qq --no-install-recommends install ca-certificates curl python3-dev gettext libcurl4-openssl-dev gcc libssl-dev
```

Avoid right-left scrolling, improve readability and sort dependancies to avoid duplicates.
```suggestion
      - run: sudo DEBIAN_FRONTEND="noninteractive" apt-get -qq --no-install-recommends install ca-certificates
                curl gcc gettext libcurl4-openssl-dev libssl-dev python3-dev 
```

---

When not using `run: |`, the second line is always interpreted as continuation of the first line? I'd have used ` \` here appended to the first line, but would be nice to know if this is redundant.

---

\ is redundant in YAML but indentation is probably significant.

This Action ___is___ testing how `make` will work so we should not take any shortcuts.

---

The question is whether we at all intend to have `make` running by any contributor or user locally or whether we want this to run via workflow in a controlled environment and have a PR with a localisation updated created by this workflow automatically as well. But until we have that workflow or change this one accordingly, I agree it makes sense to leave it as is to cover instructions we can give to anyone who wants to update localisation.

---

Let's add a comment at the top of this Action that describes the original goal of #2324